prost-build/Cargo.toml

```diff
@@ -23,7 +23,7 @@ cleanup-markdown = ["pulldown-cmark", "pulldown-cmark-to-cmark"]
 [dependencies]
 bytes = { version = "1", default-features = false }
 heck = "0.4"
-itertools = { version = ">=0.10, <0.12", default-features = false, features = ["use_alloc"] }
+itertools = { version = ">=0.10, <0.13", default-features = false, features = ["use_alloc"] }
```

Comment:
@caspermeijn one thing that might be hiding is that this version range, whether it is written as `<0.13` or `<=0.12`, allows `0.12.99999`. Some folks find that confusing with the latter version, though I also prefer to read the version that is _included_ rather than the one that is _excluded_ :+1:

Reply:
Well, the intention is to include/allow 0.12 series, so I prefer to write is the suggested way.